.vib/joomla/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+binaries:
+  - apachectl
+  - mariadb
+  - php
+  - render-template
+files:
+  - paths:
+      - /opt/bitnami/apache/modules/libphp.so
+      - /opt/bitnami/joomla/.htaccess
+      - /opt/bitnami/joomla/configuration.php
```

Comment:
You are already doing a `file` test on this file, so you can drop it from this list. Actually, it is more of a need given that Goss' logic will overwrite one of the 2 checks (the same happens if you have 2 `command` tests with the same name).

Reply:
Interesting! You are right, the count of tests didn't change after I removed this line.